libuavcan_drivers/linux/CMakeLists.txt

```diff
@@ -36,27 +42,27 @@ set(CMAKE_CXX_FLAGS "-Wall -Wextra -pedantic -std=c++0x")  # GCC or Clang
 # These aren't installed, an average library user should not care about them.
 #
 add_executable(test_clock apps/test_clock.cpp)
-target_link_libraries(test_clock ${UAVCAN_LIB} rt)
+target_link_libraries(test_clock ${UAVCAN_LIB} rt ${CMAKE_THREAD_LIBS_INIT})
 
 add_executable(test_socket apps/test_socket.cpp)
-target_link_libraries(test_socket ${UAVCAN_LIB} rt)
+target_link_libraries(test_socket ${UAVCAN_LIB} rt ${CMAKE_THREAD_LIBS_INIT})
 
 add_executable(test_node apps/test_node.cpp)
-target_link_libraries(test_node ${UAVCAN_LIB} rt)
+target_link_libraries(test_node ${UAVCAN_LIB} rt ${CMAKE_THREAD_LIBS_INIT})
 
 add_executable(test_time_sync apps/test_time_sync.cpp)
-target_link_libraries(test_time_sync ${UAVCAN_LIB} rt)
+target_link_libraries(test_time_sync ${UAVCAN_LIB} rt ${CMAKE_THREAD_LIBS_INIT})
 
 #
 # Tools
 # Someday they will be replaced with Python scripts (pyuavcan is not finished at the moment)
 #
 add_executable(uavcan_status_monitor apps/uavcan_status_monitor.cpp)
-target_link_libraries(uavcan_status_monitor ${UAVCAN_LIB} rt)
+target_link_libraries(uavcan_status_monitor ${UAVCAN_LIB} rt ${CMAKE_THREAD_LIBS_INIT})
 
 add_executable(uavcan_nodetool apps/uavcan_nodetool.cpp)
-target_link_libraries(uavcan_nodetool ${UAVCAN_LIB} rt)
+target_link_libraries(uavcan_nodetool ${UAVCAN_LIB} rt ${CMAKE_THREAD_LIBS_INIT})
 
 install(TARGETS uavcan_status_monitor
                 uavcan_nodetool
-        RUNTIME DESTINATION bin)
+        RUNTIME DESTINATION bin)
```

Comment:
Could you please add a newline here, and remove an extra empty line after `find_package(Threads REQUIRED)`.

Thanks!